### ISSUE TITLE
FOV Issue While Fishing

### ISSUE THREAD

**Issue by lovinglife17** [Lyall/RF5Fix#21]
Hello! I found an issue with the FOV that occurs when fishing. After casting the line, the FOV increases by a certain amount and keeps doing so after every cast. It goes back to normal after reeling in, but each subsequent cast keeps increasing the FOV until it reaches a point of insanity lol.

For additional context my FOV config settings are "FOVAdjustment = true" and "AdditionalFOV.Value = 15"

Screenshots:
This is after the very first cast -
![rf5 fov 1](https://user-images.githubusercontent.com/114783165/193367723-322c101a-9b94-455b-83ec-67eb736d0647.jpg)

This is after 3 or 4 casts -
![rf5 fov 2](https://user-images.githubusercontent.com/114783165/193367761-667a3667-3ec1-4e22-be9e-29038f84a09f.jpg)

This is after 7 or 8 casts -
![rf5 fov 3](https://user-images.githubusercontent.com/114783165/193367948-eace868a-8bbe-4d37-9f02-5f1c1285046f.jpg)

**Comment by formaldehydeson** [comment]
I get a similar effect to the third screenshot all the time around my farm area, it will fix itself if I rotate the camera but then it will happen again if I move around. Additional FOV set to 20

**Comment by goldenPiGames** [comment]
I have that issue too. FOVAdjustment = true and AdditionalFOV.Value = 0. I had never touched the config file before.
This is what it looks like after a while, but this took much longer than the 8 casts that @lovinglife17 did. I wasn't counting but I would say at least 25 casts.
![rf5fix_fish_fov](https://user-images.githubusercontent.com/48254127/196295791-c6b84278-c09d-4ab1-9862-871e25864652.png)

**Comment by formaldehydeson** [comment]
yeah after playing with it a bit more it seems to happen whenever the camera changes from the default eg fishing, farm view camera, locking on to stuff. Lower additional FOV value will make it a bit better/make it so it takes longer for the camera to become distorted but it will still happen eventually. Only way to reset it is by restarting game but it will again slowly get worse as you play. I just disabled FOVAdjustment completely in the config and that got rid of the issue but then you are stuck on default FOV

**Comment by dragonloverlord** [comment]
The issue that's causing this is the FOV not being reset on each cast, lock on, etc. The old FOV adjustment used in RF5Fix 0.1.0 didn't have this issue but it also didn't adapt to the scenario very well(if you set it to 65 then everything was set to FOV 65) but if that's not an issue for you then I compiled a custom build of RF5Fix 0.1.4 to use the old version of the FOV fix that works fine at least as far as I can tell anyway just make sure to follow the instruction I added to it.

https://github.com/dragonloverlord/RF5Fix/releases/tag/0.1.5

Again this is just a customized version of 0.1.4 and not an official update so YMMV and make sure to read the instruction thoroughly!
